packages/editor/src/components/media-placeholder/index.js

```diff
@@ -142,7 +142,7 @@ class MediaPlaceholder extends Component {
 		let instructions = labels.instructions || '';
 		let title = labels.title || '';
 		if ( ! instructions || ! title ) {
-			const isOneType = 1 === allowedTypes.length;
+			const isOneType = 1 === allowedTypes && allowedTypes.length;
```

Comment:
You probably wanted to do this:

```suggestion
			const isOneType = allowedTypes && 1 === allowedTypes.length;
```

Reply:
What do you think about [defining `allowedTypes`](https://github.com/WordPress/gutenberg/blob/fe51bd08684462690988de7ec61a3cdda083710a/packages/editor/src/components/media-placeholder/index.js#L134) as an empty array by default?

Reply:
Right you are on the boolean :man_facepalming: And I considered initializing `allowedTypes` to `[]` in the destructuring assignment, but then backed off because we do a `if ( ! this.props.allowedTypes )` check elsewhere. Upon reflection, since the default initialization won't mutate the value of the prop, the two approaches aren't incompatible and I think your suggestion's the way to go. Will push a commit momentarily.